Gemfile

```diff
@@ -18,7 +18,7 @@ group :test do
 end
 
 group :doc do
-  gem "redcarpet"
+  gem "redcarpet", platform: :mri
```

Comment:
Travis CI is blowing up on this line, because Rubocop for the project is requiring hash rockets. I am guessing that's a personal preference for the project, as I prefer Ruby 1.9+ colon syntax...